packages/terra-compact-interactive-list/src/utils/constants.js

```diff
@@ -34,3 +34,9 @@ export const WARNINGS = {
   COLUMN_MAX_WIDTH_UNIT_TYPE: 'columnMaximumWidth prop has different unitType than the one used in columns. It will be disregarded.',
   COLUMN_WIDTH_INCONSISTENT_TYPE: 'width, minimumWidth, and maximumWidth properties should be of the same unit type across all the columns (px, em, and rem are supported types).',
 };
+
+export const ERRORS = {
+  ROW_HEADER_INDEX_EXCEEDS_COLUMNS: 'Prop rowHeaderIndex exceeds the number of semantic columns in the row.',
+  ROW_HEADER_INDEX_LESS_THAN_ZERO: 'Prop rowHeaderIndex must be a positive integer.',
+  ROW_HEADER_INDEX_NOT_AN_INTEGER: 'Prop rowHeaderIndex must be an integer.',
```

Comment:
Could these errors be combined into a single one?

Reply:
I see your point. I was wondering the same, but as I was following the implementation in `terra-table` implemented [here](https://github.com/cerner/terra-framework/blob/main/packages/terra-table/src/utils/constants.js) I ended up keeping it the way it is for consistency.

Reply:
Ah gotcha, that makes sense